helm/qontract-reconcile/values-external.yaml

```diff
@@ -55,10 +55,10 @@ integrations:
   sleepDurationSecs: 300
   resources:
     requests:
-      memory: 150Mi
+      memory: 970Mi
```

Comment:
nit: why not just guaranteeing 1Gb of memory for our cache? Do we ever delete things from it?

Reply:
We don't delete from the cache. It's what my initial calculation gave me. And I'm pretty sure I'll end up reducing this allocation soon.

Reply:
sure, I meant why making requests and limits different... but it's only 30Mb, no biggie 😂